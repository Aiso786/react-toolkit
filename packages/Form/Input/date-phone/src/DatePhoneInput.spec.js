import React from 'react';
import renderer from 'react-test-renderer';
import DatePhoneInput from './DatePhoneInput';

describe('<DateInput>', () => {
  it('renders DatePhoneInput correctly', () => {
    const tree = renderer
      .create(
        <DatePhoneInput
          label="Image *"
          id="iddateinput"
          name="placeImage"
          onChange={() => {}}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
