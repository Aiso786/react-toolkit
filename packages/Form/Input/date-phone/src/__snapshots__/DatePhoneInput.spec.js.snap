// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DatePhoneInput> renders DatePhoneInput correctly 1`] = `
<div
  className="row af-form__group"
>
  <div
    className="col-md-2"
  >
    <label
      className="af-form__group-label"
      htmlFor="iddateinput"
    >
      Image *
    </label>
  </div>
  <div
    className="col-md-10"
  >
    <div
      className="af-form__date"
    >
      <input
        autoFocus={null}
        className="af-form__input-date"
        disabled={false}
        id="iddateinput"
        name="placeImage"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        readOnly={false}
        tabIndex={null}
        type="date"
        value=""
      />
    </div>
    <small
      className="af-form__help"
    />
  </div>
</div>
`;
